config-service/src/integrationTest/resources/configs/config-service/application.conf

```diff
@@ -2,6 +2,7 @@ main.class = org.hypertrace.config.service.ConfigService
 service.name = config-service
 service.port = 50101
 service.admin.port = 50102
+global.config.service.port.internal = 50103
```

Comment:
what's internal mean here? Also, nothing needs to be declared in here unless it's overriding the defaults - and this looks to be a duplicate

Reply:
I was following the pattern I'm guessing it means it is an internal port and not exposed externally. Will remove it from here.

Reply:
whether it's exposed or not wouldn't be decided in this config - although we do have the concept of config services meant for inside the platform and those meant for external consumers - maybe that's what you saw?

Reply:
removed it from this file